sr_linux/platform.cmake

```diff
@@ -115,10 +118,10 @@ add_definitions(
   )
 
 # Linker
-set(gtm_link  "-Wl,-u,gtm_filename_to_id -Wl,-u,gtm_zstatus -Wl,--version-script,\"${YDB_BINARY_DIR}/gtmexe_symbols.export\"")
+set(gtm_link  "-Wl,-u,gtm_filename_to_id -Wl,-u,ydb_zstatus -Wl,--version-script,\"${YDB_BINARY_DIR}/gtmexe_symbols.export\"")
 set(gtm_dep   "${YDB_BINARY_DIR}/gtmexe_symbols.export")
 
-set(libgtmshr_link "-Wl,-u,gtm_ci -Wl,-u,gtm_filename_to_id -Wl,-u,gtm_is_main_thread")
+set(libgtmshr_link "-Wl,-u,ydb_ci -Wl,-u,gtm_filename_to_id -Wl,-u,gtm_is_main_thread")
```

Comment:
I'm afraid this opened a can of worms as this help file hasn't been maintained very well. There are now so many changes in it, it will likely be a bear to merge if FIS changes it in a subsequent update.

In addition to the requested changes (which were done), I also fixed up the external call stuff which had all gtm_ types instead of ydb_ also as did the encryption description. In fact, shortly after (pre-existing) mention that the xc_* types were deprecated, they were used significantly in the encrypt section. I changed all those to ydb_ types as well.

The biggest change was 99% of the "GT.M" mentions (over 1000 instances) were changed to "YottaDB" - several of which caused significant reformatting.

There are likely some of these left in the java plugin but that's not part of this commit as its source is separate.